metricbeat/docs/running-on-docker.asciidoc

```diff
@@ -15,9 +15,9 @@ host machine from within the container.
 ["source","sh",subs="attributes"]
 ----
 docker run \
-  --volume=/proc:/hostfs/proc:ro \ <1>
-  --volume=/sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro \ <2>
-  --volume=/:/hostfs:ro \ <3>
+  --mount type=bind,source=/proc,target=/hostfs/proc,readonly \ <1>
+  --mount type=bind,source=/sys/fs/cgroup,target=/hostfs/sys/fs/cgroup,readonly \ <2>
+  --mount type=bind,source=/,target=/hostfs,readonly \ <3>
```

Comment:
These bind mounts are equivalent to the previous "volumes", and they work, but not sure if they make sense, the last one includes the other two.
In any case this is not introduced in this PR, we can continue with this in a follow-up PR.